src/wp-includes/class-wp-theme-json.php

```diff
@@ -1934,6 +1934,15 @@ protected static function compute_style_properties( $styles, $settings = array()
 			return $declarations;
 		}
 
+		// The global settings can include dynamic data related to typography. We need evaluate it so that the cache is invalidated when it changes.
+		$args      = array( func_get_args(), wp_get_global_settings() ); // phpcs:ignore PHPCompatibility.FunctionUse.ArgumentFunctionsReportCurrentValue.Changed
+		$cache_key = 'compute_style_properties_' . md5( wp_json_encode( $args ) );
+		$cache     = wp_cache_get( $cache_key, 'wp-styles' );
```

Comment:
Rather than introducing a new cache group, I think this should be a part of the existing `theme_json` group. Would need to change it below where this cache value is set, as well.

```suggestion
		$cache     = wp_cache_get( $cache_key, 'theme_json' );
```